levant/structs/config.go

```diff
@@ -43,4 +43,7 @@ type Config struct {
 	// VaiableFile contains the variables which will be substituted into the
 	// templateFile before deployment.
 	VaiableFile string
+
+	// AllowStale sets consistency level for nomad query - https://www.nomadproject.io/api/index.html#consistency-modes
+	AllowStale bool
```

Comment:
pedantic but could we move this up and keep the flags list in alphabetical order?